config/pipelines/pipelines.json

```diff
@@ -4,6 +4,10 @@
     "name": "phac-nml/iridanextexample",
     "description": "IRIDA Next Example Pipeline",
     "versions": [
+      {
```

Comment:
Can we remove the `"executable": false` from this pipeline and change one of the earlier versions to include it instead? For testing purposes we probably want the newest version to be executable